### PR TITLE
Add example table to analysis/README

### PR DESCRIPTION
**Pull request by sjspielman**
Inspired by the brief conversation in #945 about naming, I took a stab at expanding `analysis/README.md` with a table indicating which modules are provided as examples of some flavor, rather than bona fide scientific analyses.

Let me know what you think!

### CODE REVIEW

**Comment by sjspielman** [comment]
Might need to rethink this, since the `seurat-conversion` module is actually doing conversions, not necessarily demonstrating usage, which indeed we may not need to demonstrate now that we're planning on providing objects(?).

**Comment by jashapiro** [comment]
> Might need to rethink this, since the `seurat-conversion` module is actually doing conversions, not necessarily demonstrating usage, which indeed we may not need to demonstrate now that we're planning on providing objects(?).

Yeah, I don't have plans to create notebooks for Seurat conversion. It might be nice to have some demonstrations if people want to work with some of the options, but I actually think this may be better provided in the docs.

**Comment by sjspielman** [comment]
I think this README is still helpful, so I'll just rip out that line in the table.

File: analyses/README.md [hunk 1]
```diff
@@ -2,3 +2,13 @@ This directory contains the [OpenScPCA analysis modules](https://openscpca.readt
...
+
+## Example modules
+
+Several modules are provided as examples for how to use certain frameworks in OpenScPCA:
+
+| Module name | Purpose
+|-------------|---------
+| [`hello-R`](hello-R/) | Demonstrates an example structure of an R-based module
+| [`hello-python`](hello-python/) | Demonstrates an example structure of a Python-based module
+| [`hello-cluster`](hello-cluster/) | Provides notebooks demonstrating the use of clustering and cluster evaluations functions in the [`rOpenScPCA` package](https://github.com/AlexsLemonade/rOpenScPCA/)
```
Comment (jashapiro) [inline analyses/README.md:14]:
```suggestion
| [`hello-clusters`](hello-clusters/) | Provides notebooks demonstrating the use of clustering and cluster evaluations functions in the [`rOpenScPCA` package](https://github.com/AlexsLemonade/rOpenScPCA/)
```

**Review by jashapiro — APPROVED** [review]
LGTM, with typo fix. 

But now I'm a bit worried the links might not work as they are. I don't think a full url is worth doing, but I guess we can test after merge.

**Pull request merged**